app/assets/stylesheets/homes.scss

```diff
@@ -338,3 +337,12 @@ textarea {
     padding-top: 1em;
   }
 }
+
+.pruner.validate {
+  background-color: rgba(0, 0, 0, 0.4);
```

Comment:
Color literals like `rgba(0, 0, 0, 0.4)` should only be used in variable declarations; they should be referred to via variable everywhere else.